packages/instantsearch-react/src/createHOC.js

```diff
@@ -1,27 +1,26 @@
 import React, {Component} from 'react';
 import {connect} from 'react-algoliasearch-helper';
 
-import {applyDefaultProps} from './utils';
 import {stateManagerPropType, configManagerPropType} from './propTypes';
 
 const hasOwnProperty = (...args) => ({}.hasOwnProperty.call(...args));
 
 const ORIGINAL_PROPS_KEY = '__original';
 
 export default function createHOC(desc) {
-  return (mapPropsToConfig = a => a) => {
+  return (desc2 = {}) => {
```

Comment:
Or maybe `adapterDesc`, since the primary function of this parameter is to adapt the connector to a particular component's interface?